snuba/datasets/discover.py

```diff
@@ -29,7 +29,7 @@
 from snuba.query.processors.impact_processor import ImpactProcessor
 from snuba.query.processors.timeseries_column_processor import TimeSeriesColumnProcessor
 from snuba.query.project_extension import ProjectExtension, ProjectWithGroupsProcessor
-from snuba.query.timeseries import TimeSeriesExtension
+from snuba.query.timeseries_extension import TimeSeriesExtension
```

Comment:
This should probably be `snuba.query.extensions.{organization,project,timeseries}` but I didn't really feel like moving all of them right now. I can do that though if we'd prefer.